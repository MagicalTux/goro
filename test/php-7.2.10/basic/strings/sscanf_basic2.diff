     *** Testing sscanf() : basic functionality - using integer format ***
     
     -- Try sccanf() WITHOUT optional args --
004- string(6) "Widget"
005- int(1234789)
006- int(25)
     
008- -- Try sccanf() WITH optional args --
009- int(3)
010- string(6) "Widget"
011- int(1234789)
012- int(25)
013- ===DONE===
005+ Fatal error: Uncaught Error: Call to undefined function sscanf in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/sscanf_basic2.php:18
006+ Stack trace:
007+ #0 {main}
008+   thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/sscanf_basic2.php on line 18
