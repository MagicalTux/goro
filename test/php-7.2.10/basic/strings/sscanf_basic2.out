*** Testing sscanf() : basic functionality - using integer format ***

-- Try sccanf() WITHOUT optional args --

Fatal error: Uncaught Error: Call to undefined function sscanf in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/sscanf_basic2.php:18
Stack trace:
#0 {main}
  thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/sscanf_basic2.php on line 18